pdfminer/cmapdb.py

```diff
@@ -231,14 +231,14 @@ class CMapNotFound(CMapError):
     @classmethod
     def _load_data(klass, name):
         name = name.replace("\0", "")
-        filename = '%s.pickle.gz' % name
+        filename = '%s.pickle.xz' % name
         log.info('loading: %r', name)
         cmap_paths = (os.environ.get('CMAP_PATH', '/usr/share/pdfminer/'),
                       os.path.join(os.path.dirname(__file__), 'cmap'),)
         for directory in cmap_paths:
             path = os.path.join(directory, filename)
             if os.path.exists(path):
-                gzfile = gzip.open(path)
+                gzfile = lzma.open(path)
```

Comment:
The cmap files are included into the package (in the `pdfminer/cmp` directory). By changing the compression method those files cannot be used anymore.